src/main/java/frc/team997/lib/telemetry/Logger.java

```diff
@@ -20,12 +20,11 @@
 import edu.wpi.first.util.datalog.DataLog;
 import edu.wpi.first.util.datalog.DoubleLogEntry;
 import edu.wpi.first.util.datalog.StringLogEntry;
-import edu.wpi.first.wpilibj.DataLogManager;
 import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
 
 /**
- * The Logger class conveniently packages publishing telemetry data to a off-robot dashboard
- * (Shuffleboard) and saving that telemetry to an on-robot log file.
+ * Utility to streamline publishing data to NetworkTables and saving that data to an on-robot log
+ * file.
```

Comment:
Utility to streamline publishing data to NetworkTables and save it to an on-robot log file.